src/vmm/src/devices/virtio/block/mod.rs

```diff
@@ -46,11 +55,11 @@ pub enum BlockError {
     FileEngine(io::BlockIoError),
     // Error manipulating the backing file.
```

Comment:
I suppose we're not enabling the lint after all? Because it'd complain about this otherwise :thinking: